src/server.rs

```diff
@@ -1221,6 +1221,14 @@ impl ServerStats {
             }};
         }
 
+        macro_rules! set_stat_precentage {
```

Comment:
Following the style of other macros, this should be set_percentage_stat (also, you have a typo on percentage).